Dfe.Data.SearchPrototype/Web/Controllers/HomeController.cs

```diff
@@ -27,18 +32,30 @@ public class HomeController : Controller
     /// The concrete implementation of the T:DfE.Data.ComponentLibrary.CleanArchitecture.CleanArchitecture.Application.UseCase.IUseCase<SearchByKeywordRequest, SearchByKeywordResponse>
     /// defined within, and injected by the IOC container (defined within program.cs)
     /// </param>
-    /// <param name="mapper">
+    /// <param name="establishmentResultsToEstablishmentsViewModelMapper">
     /// The concrete implementation of the T:DfE.Data.ComponentLibrary.CrossCuttingConcerns.Mapping.IMapper<EstablishmentResults, SearchByKeywordResponse>
     /// defined within, and injected by the IOC container (defined within program.cs)
     /// </param>
+    /// <param name="establishmentFacetsToFacetsViewModelMapper">
+    /// The concrete implementation of the T:DfE.Data.ComponentLibrary.CrossCuttingConcerns.Mapping.IMapper<EstablishmentFacetsMapperRequest, List<Facet>?>
+    /// defined within, and injected by the IOC container (defined within program.cs) used to map all facets and pre-selections from the response to the view model.
+    /// </param>
+    /// <param name="requestMapper">
+    /// The concrete implementation of the T:DfE.Data.ComponentLibrary.CrossCuttingConcerns.Mapping.IMapper<Dictionary<string, List<string>>, IList<FilterRequest>>
+    /// defined within, and injected by the IOC container (defined within program.cs) used to map the user input to a list of <see cref="FilterRequest"/> request types.
+    /// </param>
     public HomeController(
         ILogger<HomeController> logger,
         IUseCase<SearchByKeywordRequest, SearchByKeywordResponse> searchByKeywordUseCase,
-        IMapper<SearchByKeywordResponse, SearchResultsViewModel> mapper)
+        IMapper<EstablishmentResults?, List<ViewModels.Establishment>?> establishmentResultsToEstablishmentsViewModelMapper,
```

Comment:
We have quite a mix of IList and List going on - I did changes on the train yesterday to make as much as I could see into IList - maybe best done as a separate PR after this one though